recipes-core/sysvinit/sysvinit-inittab/start_getty

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# Copyright (c) 2012-2013 National Instruments.
+# All rights reserved.
+# Spawn getty on the enabled, controlling console device. There will be at most
+# one of these. If there aren't any, don't spawn any.
+
+
+SPEED=""
+CONSOLE=""
+
+while read cons flags; do
+   case "$flags" in
+   *EC*) CONSOLE="$cons"
+         SPEED=$(stty speed < /dev/$CONSOLE)
+         echo Autodetected console "$CONSOLE" with baud rate "$SPEED"
+         break
+         ;;
+   esac
+done < /proc/consoles
+
+# If there aren't any detected consoles, exit
+[ -z "$CONSOLE" ] && exit 0
```

Comment:
It looks like the sumo version of the file defaults to tty2 in this case. Was that part no longer needed? It would've been nice to see the file as is resurrected, and then a commit on top of that modifying it.

Reply:
The commit that removed the file (https://github.com/ni/meta-nilrt/commit/f772bc9bad1ac9a49fa3f9cb22baab2ff011fedb) contained changes in another file (nilrt-runmode-system-image.postinst) that we don't want.

I could revert that commit and delete the change in that file .. but it seemed cleaner this way. Let me know if you prefer I do that.

Reply:
And regarding `tty2`, I haven't found anything that indicates its required - Bill also looked into this previously and [couldn't find](https://github.com/ni/meta-nilrt/pull/274#discussion_r751396054) anything about it either.

In console params we [only specify ttyS0](https://github.com/ni/meta-nilrt/blob/nilrt/master/hardknott/recipes-bsp/grub/grub/grub-runmode-bootimage.cfg#L3) so `tty2` doesn't even appear in `/proc/consoles` (`tty1` does appear in `/proc/consoles` even when not specified in console params but I think thats the default terminal that appears on a display connected to target).

Reply:
I just noticed that `/etc/inittab` line for serial console specifies `respawn` which means `start_getty` will be restarted if it is accidentally killed (which I guess is what we want). So in the case when console is disabled in MAX, if we return early from the script, it will be restarted continuously and we end up with following log in `/var/log/messages`

> init: Id "S0" respawning too fast: disabled for 5 minutes

I suspect `tty2` was present previously to prevent this. I'll add it back.

Reply:
Addressed this in https://github.com/ni/meta-nilrt/pull/353